velox/common/memory/MemoryArbitrator.h

```diff
@@ -81,19 +81,21 @@ class MemoryArbitrator {
   ///
   /// NOTE: we only allow the same 'kind' of memory arbitrator to be registered
   /// once. The function throws an error if 'kind' is already registered.
-  static void registerFactory(const std::string& kind, Factory factory);
+  static bool registerFactory(const std::string& kind, Factory factory);
```

Comment:
Would you update comments to explain the return value? Is the following still true?

`The function throws an error if 'kind' is already registered.`